projectDocs/translating/crowdin.md

```diff
@@ -163,15 +181,29 @@ Documentation available for translation includes:
 
 To translate any of these files:
 
-* Download the xliff file from the Files section of your language on Crowdin.
-    * Make sure to choose "Download" not "Export xliff".
-* Make a copy of this file.
+* Download the xliff file for your language from  Crowdin using NVA's l10nUtil.exe:
+```
+l10nUtil.exe downloadTranslationFile <language> <crowdinFilePath> [<localFilePath>]
+```
+E.g.
+```
+l10nUtil.exe downloadTranslationFile fr userGuide.xliff
+```
+The first time you will be asked for an authorization token.
+Please visit https://crowdin.com/settings#api-key
+And create a personal access token and paste this into the user prompt.
+this will be saved in ~/.nvda_crowdin for future use.
```

Comment:
```suggestion
Please visit [Crowdin's API page](https://crowdin.com/settings#api-key) and create a Personal Access Token.
Ensure that it has at least the translations scope.
Then paste this into the user prompt.
This will be saved in ~/.nvda_crowdin for future use.
```